datasets/lahaina-fire.data.mdx

```diff
@@ -11,10 +11,14 @@ media:
 taxonomy:
   - name: Topics
     values:
-      - EIS
+      - Disasters
+      - Burn Severity
+      - Wildfire
   - name: Source
     values:
-      - UAH
+      - HLS
+      - Landsat
```

Comment:
Does this include a Landsat data layer? I thought this was just HLS and if so, we shouldn't add Landsat here.